src/es_sfgtools/processing/assets/tiledb_temp.py

```diff
@@ -0,0 +1,188 @@
+    "east_std": tiledb.Attr(name="east_std", dtype=np.float32,nullable=True),
+    "north_std": tiledb.Attr(name="north_std", dtype=np.float32,nullable=True),
+    "up_std": tiledb.Attr(name="up_std", dtype=np.float32,nullable=True),
+    "latitude": tiledb.Attr(name="latitude", dtype=np.float32),
+    "longitude": tiledb.Attr(name="longitude", dtype=np.float32),
+    "height": tiledb.Attr(name="height", dtype=np.float32)
+}
+
+GNSSAttributes = [
+    attribute_dict["latitude"],
+    attribute_dict["longitude"],
+    attribute_dict["height"],
+    attribute_dict["east"],
+    attribute_dict["north"],
+    attribute_dict["up"],
+    attribute_dict["east_std"],
+    attribute_dict["north_std"],
+    attribute_dict["up_std"],
+    tiledb.Attr(name="number_of_satellites", dtype="uint8"),
+    tiledb.Attr(name="pdop", dtype=np.float32),
+]
+GNSSArraySchema = tiledb.ArraySchema(
+    sparse=True,
+    domain=tiledb.Domain(TimeDomain),
+    attrs=GNSSAttributes,
+    cell_order='col-major', 
+    tile_order='row-major',
+    allows_duplicates=False,
+    coords_filters=filters,
+)
+
+PositionAttributes = [
+    tiledb.Attr(name="head", dtype=np.float32),
+    tiledb.Attr(name="pitch", dtype=np.float32),
+    tiledb.Attr(name="roll", dtype=np.float32),
+    attribute_dict["east"],
+    attribute_dict["north"],
+    attribute_dict["up"],
+    attribute_dict["east_std"],
+    attribute_dict["north_std"],
+    attribute_dict["up_std"],
+
+]
+PositionArraySchema = tiledb.ArraySchema(
+    sparse=True,
+    domain=tiledb.Domain(TimeDomain),
+    attrs=PositionAttributes,
+    cell_order='col-major', 
+    tile_order='row-major',
+    allows_duplicates=False,
+    coords_filters=filters,
+)
+
+ShotDataAttributes = [
+    tiledb.Attr(name="head0", dtype=np.float32),
+    tiledb.Attr(name="pitch0", dtype=np.float32),
+    tiledb.Attr(name="roll0", dtype=np.float32),
+    tiledb.Attr(name="head1", dtype=np.float32),
+    tiledb.Attr(name="pitch1", dtype=np.float32),
+    tiledb.Attr(name="roll1", dtype=np.float32),
+    tiledb.Attr(name="east0", dtype=np.float32),
+    tiledb.Attr(name="north0", dtype=np.float32),
+    tiledb.Attr(name="up0", dtype=np.float32),
+    tiledb.Attr(name="east1", dtype=np.float32),
+    tiledb.Attr(name="north1", dtype=np.float32),
+    tiledb.Attr(name="up1", dtype=np.float32),
+    attribute_dict["east_std"],
+    attribute_dict["north_std"],
+    attribute_dict["up_std"],
+]
+
+ShotDataArraySchema = tiledb.ArraySchema(
+    sparse=True,
+    domain=tiledb.Domain(TimeDomain),
+    attrs=ShotDataAttributes,
+    cell_order='col-major', 
+    tile_order='row-major',
+    allows_duplicates=False,
+    coords_filters=filters,
+)
+
+accousticTimeDim = tiledb.Dim(name="triggerTime", domain=(0,), tile=100, dtype="datetime64[ms]")
+acousticIDDim = tiledb.Dim(name="transponderID", domain=(0,), tile=100, dtype="U")
+
+AcousticDataAttributes = [
+    tiledb.Attr(name="transponderID",dtype="U"),
```

Comment:
if this is a dimension, it does not also need to be an attribute

Reply:
trying to think about the value of this as a dimension vs as an attribute.  do we ever want to query all data for a single transponder? if so, it makes sense to be a dimension